configs/platforms/debian-12-armhf.rb

```diff
@@ -0,0 +1,30 @@
+platform "debian-12-armhf" do |plat|
```

Comment:
How is this file difference from `configs/platforms/debian-12-aarch64.rb`
Can you share why do we need this file as for previous debian platforms we dont have this file.

Reply:
This is for the ARMv7 architecture (eg. raspberry pi?)

Reply:
The platforms are also there for debian-10 and debian-11.